.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
It honestly would be super helpful for me. I would love to not do a hostile fork and continue contributing to the same source instead.